frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/models/vms/VmListModel.java

```diff
@@ -787,6 +788,28 @@ private void edit() {
 
     }
 
+    private void confirmChangedBiosType() {
```

Comment:
Suggestion (not sure if better): `confirmChangedBiosType` --> `confirmBiosTypeUpdate`.

Reply:
Everything has been fixed as posted.